collector/engine_tokudb_test.go

```diff
@@ -0,0 +1,69 @@
+package collector
+
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/smartystreets/goconvey/convey"
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"
+)
+
+func Test_sanitizeTokudbMetric(t *testing.T) {
```

Comment:
Do you want to remove the underscore here and below while you're at it, to make it more Go-idiomatic?